torch/ao/quantization/fx/backend_config/utils.py

```diff
@@ -101,3 +101,22 @@ def get_root_node(node_pattern):
             root_node_getter_mapping[pattern] = root_node_getter
 
     return root_node_getter_mapping
+
+def get_fusion_pattern_to_extra_inputs_getter(
+        backend_config_dict: Dict[str, Any]) -> Dict[Pattern, Callable]:
+    """ Get a map from fusion pattern to a function that returns extra input nodes
+    from the fusion pattern, in the order required by the root node. This is optional,
+    if not specified, we will not copy over any extra inputs for the root node.
+    Example:
+    # given a pattern, return the input node that need to be copied to the root node
```

Comment:
can we add a more specific example?  It's hard to understand what this means from just the docblock

Reply:
yeah sure